recipes/nanort/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.15)
+
+project(test_package LANGUAGES CXX)
+
+find_package(nanort REQUIRED CONFIG)
+
+add_executable(${PROJECT_NAME} test_package.cpp)
+target_link_libraries(${PROJECT_NAME} PRIVATE nanort::nanort)
```

```suggestion
target_link_libraries(${PROJECT_NAME} PRIVATE nanort::nanort)
target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_11)

```
This is needed for our v1 pipeline, because we don't pass cppstd to the build, but in v2 would not be necessary as it's already set by Conan coming from the profile.